motr/linux_kernel/m0tr_main.c

```diff
@@ -64,7 +71,8 @@ module_exit(motr_exit);
  * marker is necessary to remove the warnings, keeping this blank to make
  * compiler happy.
  */
-MODULE_LICENSE();
+/* Added GPL per suggestions to avoid compilation error, may need to be reviewed */
+
```

Comment:
Why is this comment left? And why `MODULE_LICENSE();` is removed?

Reply:
Removed and reverted